lib/reducers/auth/store_user.dart

```diff
@@ -1,11 +1,18 @@
+import 'package:built_collection/built_collection.dart';
```

Comment:
this was a fix for a bug where the page on top of the InitialPage does not pop after sign in

Reply:
and pop she will, nice!